XLS-0026-iou-token-metadata/README.md

```diff
@@ -1,8 +1,12 @@
 <pre>
-Title:       <b>IOU Token Metadata via xrp-ledger.toml</b>
-Revision:    <b>5</b> (2025-06-06)
-Author:      <a href="https://github.com/Mwni">Marc-Emanuel Otto</a>
-Contributor: <a href="https://ripple.com">Ripple</a>
+xls: 26
+title: IOU Token Metadata via xrp-ledger.toml
+description: A unified solution for IOU token metadata by adding optional fields to the existing xrp-ledger.toml standard
+author: Ed Hennis <ed@ripple.com>
+discussion-from: https://github.com/XRPLF/XRPL-Standards/discussions/26
```

Comment:
This is the wrong link - it shouldn't be pulling the actual Github discussion number, but the discussion that has the number in its title. Ditto for all the other XLSes.